.github/workflows/client.yml

```diff
@@ -28,19 +28,25 @@ jobs:
       - name: Run the linting checks
         run: |
           cd client/	
-          yarn install	
+          yarn install
           yarn syntax
       - name: Build the React SPA website
         run: |
           cd client/	
-          yarn install	
+          ./script/install.bash 
           yarn build
           yarn configapp dev
       - name: Run client unit and integration tests
         run: |
           cd client/	
-          yarn install	
+          yarn install
           yarn test
+      - name: 'Upload Artifact'
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: E2E-error-trace
+          path: client/test-results
```

Comment:
It should upload to the [summary of the job](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts). I didn't manage to only run if the previous job failed, hence I'm using `always()`. The advantage is that the trace can be activated with a simple playwright configuration and uploaded even if the job is not failing if the developer is interested.

here's an [example job](https://github.com/Omarley7/DTaaS-Bachelor-new-GUI/actions/runs/5305497501) with artifacts uploaded.

Reply:
Good, I am after the file you have shown in [here](https://github.com/Omarley7/DTaaS-Bachelor-new-GUI/actions/runs/5305497501). This file was not generated for this PR. Is it only generated for commits and not PRs?

Reply:
Hmm, that is very strange. I see no reason why it shouldn't produce artifacts for a PR. Do you have a link where I can see the PR job?

Reply:
https://github.com/INTO-CPS-Association/DTaaS/actions/runs/5305452168

Reply:
Since the Action has no failing tests, there's no error trace uploaded. I've removed the warning by only attempting to upload if the previous job (tests) fails.